### PR TITLE
Enable snapshots

### PR DESCRIPTION
**Pull request by freshe4qa**
Enable snapshots to enable validators to more quickly deploy nodes/run experiments

## Description

closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by robert-zaremba — COMMENTED** [review]
Thanks for contribution. Could you update the Changelog please?

**Comment by freshe4qa** [comment]
>

**Comment by freshe4qa** [comment]
@robert-zaremba done

**Comment by robert-zaremba** [comment]
@freshe4qa did you push the changes? I don't see the changelog update.
Also, could you look at the failing tests?

**Comment by freshe4qa** [comment]
@robert-zaremba i fix lint, check changes

**Comment by robert-zaremba** [comment]
still the test and lint is failing.

**Comment by freshe4qa** [comment]
Everyone has different Lint and other things set up. I won't fix anything there so as not to ruin what's already working. I think it is better for you to look at what can be fixed or someone from the team.

File: cmd/umeed/cmd/root.go [hunk 1]
```diff
@@ -241,3 +244,21 @@ func txCommand(ac appCreator) *cobra.Command {
 
 	return cmd
 }
+
+// Add snapshots
```
Comment (robert-zaremba) [inline cmd/umeed/cmd/root.go:248]:
```suggestion
	// Add snapshots
```

**Review by robert-zaremba — APPROVED** [review]
lint fix

**Comment by robert-zaremba** [comment]
You should use `make lint` to make sure you use the same config as everyone in the repo. You won't need to install anything.

**Review by robert-zaremba — CHANGES_REQUESTED** [review]
in fact this seams to be a copy paste or a commit cherry pick and it's not compatible with the file,... that's why the build fails.

**Comment by robert-zaremba** [comment]
snapshot is already implemented in https://github.com/umee-network/umee/blob/main/cmd/umeed/cmd/app_creator.go#L64

**Pull request closed without merging**